lib/main/STM32G4/Middlewares/ST/STM32_USB_Device_Library/Class/MSC/Src/usbd_msc.c

```diff
@@ -297,6 +297,17 @@ static DMA_DATA_ZERO_INIT USBD_MSC_BOT_HandleTypeDef ClassData;
 uint8_t  USBD_MSC_Init (USBD_HandleTypeDef *pdev, uint8_t cfgidx)
 {
   UNUSED(cfgidx);
+  USBD_MSC_BOT_HandleTypeDef *hmsc;
```

Comment:
@SteveCEvans: I tried to rebase this, but it seems to be re-introducing code that you removed in https://github.com/betaflight/betaflight/pull/10926 - can you please have a look?

Reply:
The #10926 libraries should be used. The `hmsc` definition is not required.